docs/api/browser-window.md

```diff
@@ -580,6 +580,15 @@ Returns:
 
 Emitted when the window is set or unset to show always on top of other windows.
 
+#### Event: 'visible-on-all-workspaces-changed'
```

Comment:
Since the tests run iff platform != win32, does this line need `_macOS_ _Linux_` ?

Reply:
no - this runs on all platforms but i believe CI makes this unreliable :)